nodejs/awsx/ec2/vpc.ts

```diff
@@ -504,23 +504,31 @@ utils.Capture(Vpc.prototype).initializeVpcArgs.doNotCapture = true;
 async function getAvailabilityZones(
         parent: pulumi.Resource | undefined,
         provider: pulumi.ProviderResource | undefined,
-        requestedCount: "all" | number | undefined): Promise<topology.AvailabilityZoneDescription[]> {
+        requestedZones: VpcArgs["requestedAvailabilityZones"] = 2): Promise<topology.AvailabilityZoneDescription[]> {
```

Comment:
But this type is no longer valid now, right? Won't this return a `Promise<Output<AvailabilityZoneDescription[]>>` with the code you have below? I think you'll want to actually want this function to return Output<AvailabilityZoneDescription[]> and then plumb that through the rest of the code?

Reply:
> But this type is no longer valid now, right? Won't this return a `Promise<Output<AvailabilityZoneDescription[]>>` with the code you have below? 

Are you sure about this? Afaik the type of the `Promise<T>`is whatever is within the resolve, which is a `topology.AvailabilityZoneDescription[]`. 
That, and TS doesn't seem to complain, but I am not sure what goes on with `Output`. 
The type *should* be `Promise<topology.AvailabilityZoneDescription[]>` as the resolve callbacks with the list, we don't return the output itself, just the resolved value. 

>  I think you'll want to actually want this function to return `Output<AvailabilityZoneDescription[]>` and then plumb that through the rest of the code?

Sure, but that would introduce breaking changes, and I quite understand the plumbing between `Input/Promise/Output`, in a side note, I feel like `Output` should at least behave like a `PromiseLike<T>` but that's a different topic. 


Regardless, do you see an actual issue running the test suite or in your compiler? Because I don't see any errors here, and I don't see why there would be since the type would be `Promise<topology.AvailabilityZoneDescription[]>`

Reply:
@leezen Could you run the tests, or give me the permissions to run them for the duration of this PR, so I could have some faster feedback loop.